README.md

```diff
@@ -14,6 +14,8 @@ Griffel ([_German for stylus/pen_](https://en.wiktionary.org/wiki/Griffel)) is C
 
 📝 &nbsp; _Experimental_ CSS extraction with [Webpack plugin](./packages/webpack-extraction-plugin)
 
+🐞 &nbsp; Debug using [griffel devtools extension](https://chrome.google.com/webstore/detail/griffel-devtools/bejhagjehnpgagkaaeehdpdadmffbigb)
```

Comment:
```suggestion
🐞 &nbsp; Debug using [Griffel DevTools extension](https://chrome.google.com/webstore/detail/griffel-devtools/bejhagjehnpgagkaaeehdpdadmffbigb)
```

Do you want to include your video there?

Reply:
Heyy there's an option to add a video for chrome extension:
<img width="758" alt="image" src="https://user-images.githubusercontent.com/28751745/179224520-a105dcfa-d715-498f-86b0-4349b7c70f2f.png">
I'll give this a try first :)